src/components/MainSidebar/ProgramDimensionsPanel/useProgramDimensions.js

```diff
@@ -99,7 +99,7 @@ const createDimensionsQuery = ({
     ) {
         // This works because data element IDs have the following notation:
```

Comment:
I'm not sure about this comment.
The format we should store the dimension (at least in `current`) needs to be:
`{
dimension: [dimensionId],
programStage: { id: [programStageId] }
}`